### PR TITLE
Added script for collecting svg icons

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Internal: Added script for collecting SVG icons from CKEditor 5 packages.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Comment by pomek** [comment]
@godai78, could you verify we understood the goals? Instruction on what to do with this script, you can find the issue.

**Comment by godai78** [comment]
We like it and we want it. Internal so far, but we want to revisit the idea and maybe make it public in the future as an UI creation aid.  
  
You can go on and merge it now. Can we have this available as a static page somewhere?

**Comment by pomek** [comment]
> Can we have this available as a static page somewhere?

So far, it will be available only locally. I would not spend much time publishing it. If I understood correctly, we would add it somewhere in the documentation.

**Comment by godai78** [comment]
Actually, we need it more in Notion now, than the docs themselves.

File: scripts/collect-svg-icons.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	return [
+		'<!DOCTYPE html>',
+		'<html>',
+		'	<head>',
+		'		<style>',
+		'			body { font-family: arial }',
+		'			table { border-collapse: collapse; margin: auto; margin-bottom: 25vh; }',
+		'			thead tr:first-child th { padding-top: 20px; font-size: 150%; text-transform: uppercase; }',
+		'			td { border: 1px solid black; }',
+		'			img { max-height: 20px; }',
```
Comment (pomek) [inline scripts/collect-svg-icons.js:86]:
```suggestion
		'			* { box-sizing: border-box; }',
		'			body { font-family: Arial; }',
		'			table { border-collapse: collapse; margin: auto; margin-bottom: 25vh; }',
		'			thead tr:first-child th { padding-top: 20px; font-size: 150% }',
		'			tbody tr:nth-child(odd) td {background: #e5e5e5;}',
		'			td { border: 1px solid black; padding: 2px; }',
		'			img { max-height: 20px; }',
```

image

**Review by pomek — CHANGES_REQUESTED** [review]
As mentioned, there are a few visual changes and some changes in the code (formatting + naming).

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**